doc/application_plugins/rlm_rest.rst

```diff
@@ -0,0 +1,65 @@
+.. _rlm_rest:
+
+Configuration of rlm_rest
+=========================
+
+Starting with version 3.0.x FreeRADIUS is shipped with the ``rlm_rest`` module, which can be used to transform
+RADIUS authentication requests to HTTP requests to a suitable REST endpoint. Starting with version 2.19,
+privacyIDEA implements such an endpoint (``/validate/radiuscheck``, see :ref:`rest_validate`).
+
+Please note that the endpoint currently does not implement all features of the :ref:`rlm_perl` such as
+challenge-response authentication and attribute mapping.
+
+On Ubuntu 17.04, FreeRADIUS and ``rlm_rest`` can be installed as follows::
```

Comment:
Yep, Ubuntu 17.04 ships FR 3.0.12 and also the ``freeradius-rest`` package: http://packages.ubuntu.com/zesty/freeradius-rest